src/PlaydateForm/PlaydateForm.js

```diff
@@ -23,13 +23,23 @@ class PlaydateForm extends Component {
   handleSubmit = (event) => {
     event.preventDefault()
     const { date, location } = this.state
-    const { currentUserId, selectedUserId, addPlaydateForSelectedUser } = this.props
+    const { currentUserId, selectedUserId, updateProfile, updateCurrentUser } = this.props
     const usersInvolved = [ currentUserId, selectedUserId ]
-    const newPlaydate = {
-      id: Date.now(),
-      date, location, usersInvolved
-    }
-    addPlaydateForSelectedUser(newPlaydate)
+    const playdateId = Date.now()
+    const playdates = usersInvolved.map(id => {
+      return {
+        id: playdateId,
+        userId: id,
+        playmate: usersInvolved.find(userId => userId !== id),
+        date, location,
+      }
+    })
+
+    fetchCalls.postAppointment(playdates)
```

Comment:
Is there any form validation before the form is submitted? You don't want to be POSTing data if the form is invalid.